cmd/keytransparency-delegate/main.go

```diff
@@ -12,12 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package main is a delegate server that can be used to
+// (a) create user accounts.
+// (b) update user accounts (that this server has created).
+//
+// The delegate server is desiged to be used by app operators
+// to provision and update users before users take control over
+// their own key management.
+//
+// The delegeate server may also be used to implement a third_party account
```

Comment:
typo: delegate
also: third party

Reply:
done

Reply:
amazing how I think of `third_party` as a whole word now :-)